### ISSUE TITLE
[Dev Branch] Anime tag not being applied to show lists when adding new anime

### ISSUE THREAD

**Issue by XxUnkn0wnxX** [pymedusa/Medusa#8418]
**Describe the bug**
just noticed this bug, i believe about 4-7 days ago its been fine but i couldn't of know any sooner as i dont add new anime regularly.
every time i add a new anime show, it doesn't apply the ANime tag in the shows list & instead applies the Series tag instead.

**To Reproduce**
just try adding a new anime show
EG: Suppose a Kid From the Last Dungeon Boonies Moved to a Starter Town
& apply the Anime option & then you will see it doesnt show in the Anime List

**Expected behavior**
to show in anime list

**Screenshots**
![Screen Shot 2020-08-31 at 5 45 49 pm](https://user-images.githubusercontent.com/2242880/91696497-01cb5e80-ebb3-11ea-9966-988e2ec56d36.jpg)
![Screen Shot 2020-08-31 at 5 46 37 pm](https://user-images.githubusercontent.com/2242880/91696515-098b0300-ebb3-11ea-87ba-02817bfa2780.jpg)

**Medusa (please complete the following information):**
 - OS: macOS Catalina 10.15.6
 - Branch: develop
 - Commit: cd208bfa8cae8670e0f33262da090b84f6ea7df6
 - Python version: 3.8.5 
 - Database version: 44.16

**Debug logs (at least 50 lines):**
[application.log.zip](https://github.com/pymedusa/Medusa/files/5149037/application.log.zip)

**Additional context**
i did have issues with show lists not displaying any shows a while back, i had to delete them from General Config --> Interface
but i did Re-add them.
![Screen Shot 2020-08-31 at 5 57 23 pm](https://user-images.githubusercontent.com/2242880/91696776-71d9e480-ebb3-11ea-9fcb-9e874269750e.jpg)

and i do have the Look & Feel Anime Option Enabled:
![Screen Shot 2020-08-31 at 6 11 12 pm](https://user-images.githubusercontent.com/2242880/91698020-6b4c6c80-ebb5-11ea-9cd9-c14987f70284.jpg)

**Comment by deusxanime** [comment]
Thanks for opening this, I'm having the same problem as well. 

Also it doesn't seem to be automatically turning on the "Scene Numbering" for anime either, which I think it did before? I think the ones that have Xem entries do this? At least I've never manually enabled it before but I see it turned on for many of the previous anime I've added in the past before whatever update it was that started causing these issues. But new ones I've added don't enable it automatically and just give me a warning in the log and recommend enabling it. Is that the new behavior expected going forward or did something break/change?

```
2020-09-10 11:43:22 WARNING  SHOWQUEUE-ADD :: [] 381406: while adding the show Uzaki-chan Wants to Hang Out! we noticed thexem.de has an episode mapping available
you might want to consider enabling the scene option for this show.
```

![image](https://user-images.githubusercontent.com/11891866/92764318-efdb7f80-f359-11ea-94fe-f86a1343d1fa.png)

Running v0.4.3 on unRAID docker container

**Comment by medariox** [comment]
It was changed because some people complained about it. It has to be turned on manually now.

**Comment by deusxanime** [comment]
> It was changed because some people complained about it. It has to be turned on manually now.

OK good to know. I'll have to pay more attention to that from now on. At least with the warning it will remind me.

Thanks!

**Comment by p0psicles** [comment]
The auto xem has been removed. We replaced it with a warning (and notification) when you add a show that has xem numbering. It was causing issues for users who didn't want to use xem numbering

**Comment by p0psicles** [comment]
fixed in #8448
